snapcraft/models/project.py

```diff
@@ -787,6 +788,26 @@ class App(models.CraftBaseModel):
     reference for more information.
     """
 
+    success_exit_status: list[Annotated[int, Ge(1), Le(255)]] | None = pydantic.Field(
+        default=None,
+        description="The list of exit codes that are considered successful.",
+        examples=["[42, 250]"],
+    )
+    """The list of exit codes that are considered successful.
```

Comment:
What do you think about:

```suggestion
        description="The list of additional successful exit statuses that 
        the service can return.",
        examples=["[42, 250]"],
    )
    """The list of additional successful exit statuses that the service can
    return.
```